doc-src/install.rst

```diff
@@ -31,287 +336,990 @@ is currently tested only on x86_64 and ARM.
-* :code:`ch-build2dir`: Bash 4.1+
+  * C11 compiler (but not Intel CC)
 
-Buildah (privileged or unprivileged)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Bootstrap build from Git:
 
-Charliecloud uses Buildah's "rootless" mode and ignore-chown-errors storage 
-configuration to acheive a fully unprivileged workflow. If an image with 
-multpile user mappings is pulled using this method, all file permissions will
-be remapped to a single user (root). This requires Buildah v1.10.1 or newer. 
+  * Git
+  * GNU Autotools
 
-For a privileged workflow, Charliecloud can also use Buildah with setuid
-helpers :code:`newuidmap` and :code:`newgidmap`. This will not remap the file 
-permissions to root.
+Re-build documentation:
 
-.. note::
+  * Python
+  * Sphinx
 
-   To configure Buildah in rootless mode, which is what Charliecloud uses,
-   make sure your config files are in :code:`~/.config/containers` and they
-   are correct. Particularly if your system also has configuration in
-   :code:`/etc/containers`, problems can be very hard to diagnose.
+Build test suite:
 
-..
-   For example, with different mistakes in
-   :code:`~/.config/containers/storage.conf` and
-   :code:`/etc/containers/storage.conf` present or absent, and all in rootless
-   mode, we have seen various combinations of:
+  * Python
 
-     * error messages about configuration
-     * error messages about :code:`lchown`
-     * using :code:`storage.conf` from :code:`/etc/containers` instead of
-       :code:`~/.config/containers`
-     * using default config documented for rootless
-     * using default config documented for rootful
-     * exiting zero
-     * exiting non-zero
-     * completing the build
-     * not completing the build
+Note: Built documentation is included in the tarballs.
 
-   We assume this will be straightened out over time, but for the time being,
-   if you encounter strange problems with Buildah, check that your config
-   resides only in :code:`~/.config/containers` and is correct.
+Details
+-------
 
-:code:`ch-grow` (unprivileged)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+There are more details for some of the dependencies; these are listed below.
 
-This is our internal unprivileged image builder and requires:
+C11 compiler
+~~~~~~~~~~~~
 
-* `skopeo <https://github.com/containers/skopeo>`_
-* `umoci <https://github.com/openSUSE/umoci>`_
-* Python module :code:`lark`
-  (`lark-parser <https://pypi.org/project/lark-parser/>`_ on PyPI)
+We test with GCC. Core team members use whatever version comes with their
+distribution.
 
-Docker (privileged)
-~~~~~~~~~~~~~~~~~~~
+In principle, any C11 compiler should work. Please let us know any success or
+failure reports.
+
+Intel :code:`icc` is not supported because it links extra shared libraries
+that our test suite can't deal with. See `PR #481
+<https://github.com/hpc/charliecloud/pull/481>`_.
+
+GNU Autotools
+~~~~~~~~~~~~~
+
+.. todo::
+
+   Do we want to say anything here? What specifically do people need to
+   install?
```

Comment:
For, RHEL systems. They need to install: 
`automake autoconf libtool`. 

I have yet to do an autotools build on Python. I'll try it out and report back.